sql/tests: TestRandomSyntaxFunctions failed [error in crdb_internal.inject_hint]

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20803026?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20803026?buildTab=artifacts#/) on master @ [1b46be83de2eb3ca447c3dc02f6b28c005526e6a](https://github.com/cockroachdb/cockroach/commits/1b46be83de2eb3ca447c3dc02f6b28c005526e6a):


```
    rsg_test.go:125: Last executed (4): SELECT array_prepend('03:49:31.019042':::TIME, NULL) 
    rsg_test.go:125: Last executed (5): SELECT array_prepend('13:18:00.050859+07:07':::TIMETZ, ARRAY[]:::TIMETZ[]) 
    rsg_test.go:125: Last executed (6): SELECT array_prepend('{}':::JSONB, ARRAY['["Qs0", true]':::JSONB,'{"foobar": 1.530732705190491}':::JSONB]) 
    rsg_test.go:125: Last executed (7): SELECT array_prepend(B'110000011011001100100001011011011010111101111110111111000001001101', ARRAY[B'0000110110000101001110100010001110000100000010110110',B'0111111111111111111111111111111111111111111111111111111111111111',B'00111010111111101001000',B'1001100000100010111010110011000011100101100011001000011000001100001100110110101110100000']) 
    rsg_test.go:125: Last executed (8): SELECT array_prepend((SELECT NULL)::ltree, ARRAY['ljtUB16l.5n':::LTREE,'hlbw.tdDQfLA.0LS-.hTdoe._dH.UsFmVE.fW.n5cIDir.3iG':::LTREE,'zmNXkdDSz.H6w.FtFw.x_FFFq.X':::LTREE]) 
    rsg_test.go:125: Last executed (9): SELECT array_prepend(NULL, ARRAY[NULL,NULL,NULL]:::anyenum[]) 
    rsg_test.go:125: Last executed (10): SELECT array_prepend((1605999286:::INT8,), ARRAY[(e'\x06':::STRING,)]) 
    rsg_test.go:125: Last executed (11): SELECT array_recv(0.8786878164311596958:::DECIMAL) 
    rsg_test.go:125: Last executed (12): SELECT array_remove(ARRAY[true,false,false,false,true], false) 
    rsg_test.go:125: Last executed (13): SELECT array_remove(ARRAY[]:::BOX2D[], 'BOX(-0.8069895419128288 -1.1043936856742163,-0.018420831069558607 -0.8842691574559244)':::BOX2D) 
    rsg_test.go:125: Last executed (14): SELECT array_remove(ARRAY[(-2147483648):::INT8,9186505844856389796:::INT8], (-9078749380237451090):::INT8) 
    rsg_test.go:125: Last executed (15): SELECT array_remove(ARRAY['NaN':::DECIMAL,(-7.162288369716800147E+32):::DECIMAL,(-3.340900288317873762E+21):::DECIMAL,(-3.500356069107535116E+20):::DECIMAL], 8.991632639180775750E+19:::DECIMAL) 
    rsg_test.go:125: Last executed (16): SELECT array_remove(ARRAY[]:::FLOAT8[], (-0.04279376371966759):::FLOAT8) 
    rsg_test.go:125: Last executed (17): SELECT array_remove(ARRAY['1993-07-05':::DATE,'1984-09-06':::DATE,'1994-01-16':::DATE], '1986-04-23':::DATE) 
    rsg_test.go:125: Last executed (18): SELECT array_remove(ARRAY['-67 years -10 mons -476 days -00:20:55.867427':::INTERVAL,'18 years 5 mons 214 days 16:28:43.523906':::INTERVAL], '48 years 6 mons 524 days 08:43:32.07094':::INTERVAL) 
    rsg_test.go:125: Last executed (19): SELECT array_remove(ARRAY['0102000020E6100000020000000013933801DF594040004F3B6CFC26C030DDFFB4CE196140F0DB9BDBF5822AC0':::GEOGRAPHY,'0102000060E6100000020000006838E1A60F5943406888723534A444406B86C9E853B8FAC1E03D7A6AB60C2C4068AB72885B2046402467918A93D0F341':::GEOGRAPHY,'0107000020E610000000000000':::GEOGRAPHY,'01060000E0E61000000500000001030000C001000000060000000017E84FDACF5CC0DE918CE01DC33EC02A70A36C63E8F941B67F82D51513014264FFF8D146375C40C0851F402DDE0440D441925F8DC2F541AA6DDBF0C288F341CCFA0655FA935B40C04DF8EBA1774B4068E8BA778101FF41086816ABA939DF41908FCAAD32DF524050E22284CC9F414084CED488AA87F2413236DBA3FEA2F7418822277FA06B544050868D83F199464058649413F4BBDA41A0F2518F53B6FBC10017E84FDACF5CC0DE918CE01DC33EC02A70A36C63E8F941B67F82D51513014201030000C00100000007000000486B10E2EE0F36C0570271F07EDB41C0766D8898DD5EF741F05EA5BF3541FD413828BB50085643C026C97DDB6E5A47C0209B43168B2A014256F1E082F683F8C1303FC81F4B96574007A2A4AB71B250C080ECA1A0B6FA924190761C97AD26F54118E560F5FA126340BD9B9D19164947C02C40D4C0A30BF7C174E453878835F241598102E1D0265FC050B5E404CCD52940F8787EA92C52DD418EED9A90AD0FF841E6BBBF45A85B57C0DCFB2CA87E2C35C00CEB82E61187E041324B0CC3426FE5C1486B10E2EE0F36C0570271F07EDB41C0766D8898DD5EF741F05EA5BF3541FD4101030000C00100000008000000D0BBBA8848AB2A40D8348153F82825C0E8B973061101E041582B79519D22F5419A80994F515C57C05A8DDEBBCB9252C0448BD8680882FF41C80C53BE57DAFEC13011E59591DC41402B307AC9339447C0B6B63DA1AC6DF8C1FCCCA3D97C6701C2588F9A06474F4F407C93BF39D58043C0B2935D00E3E1FDC114359A729E59F04152D262F7C2FA65404068F816466812C0C8E5BE7D975CD741DA33D232EA79EBC1A40917B718DA6540B0A162B1331C5440D8E6823F80810242FC3F856AE0E5F5C12C22CFB78BE753C0D0A92C8898514940806462CC9F91FAC17071524A512FD541D0BBBA8848AB2A40D8348153F82825C0E8B973061101E041582B79519D22F54101030000C0010000000C00000096E2A792C9A257C0E05487F28A553C4016B69FD436F0F4C170F8CC8C3A1EC94172ECFA6DA24163C06C38B5F3BCC7384018527D3569AFFD41A4FC921BCC3D014240FA90C7D11C1F40C4F3589DBD7833C0D024B27ED22E0142BC4AE52DB0C1EEC11061A668E48B5B4046F3B5B761A440C002CBE706D05CF841C0A0B8872CC5B8C100B80F76AB3362400010303F55C0973F984857CCA913FC4150B11558034BC541522DE426031062402875B4E3F8664240C467F54BEE62EBC198FD012C09D0D54104E36F9FAC7465403C1AFFC365334540C08405E951F6AD41ACD7613BF7FBFD41566F75CCD0286140EE05B6F2F51C5640608C9258CB7400C264D71183B20BD5C1BC5E924BDEB95040384DCA83031C4E40E032F19A36A6D0C14AC1E3D43F3501C2AC97F7A8E0684B405AEF8141A9D05240749947A8D2E3E641BA12AAAD4E93F241750F2D05126465C030225362A0DC51400AEEC2382DF700C21CA2CFFDCCC6F74196E2A792C9A257C0E05487F28A553C4016B69FD436F0F4C170F8CC8C3A1EC94101030000C0010000000A000000546B4CA844B960C00861F829BB7537402A4A04C1DA50F841908E56C1DBA0CFC1F660D2C2BB294BC09C7A56BC20D83240D0487586967AD94168559C47E129EC41E845D5C186933740A876F3A773763540082CF3C6510601C2D0A4DAB584D6CBC17815C6A1B7F347C0D8CF90B0B2B653C00C3B473136590242C8DB838A93A1DB41DCE6F7A5A78259400432AFB0101F3140C46350858829FFC1B08EEFDDFF7FEAC1187DC517654B6440C290B2C3E0BF51404E592FDD19990242203BCFBC3610BDC1FC2C1C8A89585B4030F7C3FE93434C40A19AB5F3FD73FDC11077217EF4CCF241142F07CF10CC4740305E6F141B3E4740AD0927C0536501C2C0468056130FBE41347425D137394740A686B0D38A7B514090111A01F8C90142C0D760442B9BC741546B4CA844B960C00861F829BB7537402A4A04C1DA50F841908E56C1DBA0CFC1':::GEOGRAPHY], (SELECT NULL)::geography) 
    rsg_test.go:125: Last executed (20): SELECT array_remove(ARRAY[]:::TIMESTAMP[], NULL) 
    rsg_test.go:125: Last executed (21): SELECT array_remove(ARRAY['010400004001000000010100004080880B733511F041BCA5BD41E622FE4110872358127DF5C1':::GEOMETRY,'0104000080020000000101000080068B7C957587FEC177383BF41B3402C2C41C292572DDF3C10101000080BC4FE6EB2F2900C244545ABD53D3E7413651E7D7FD5AF7C1':::GEOMETRY,'01030000C00100000006000000BBD977E7C3CCF2C136DE800E30BFE2C11FFA3CE1D1B3F1C1D6649648081CEFC186A6B55737B8FA41328C805093E9E3C1801667A37A049341C037790F525EBD415433BFB637D6F341402198726F1DBD4124FDB082FC97F6C1E6A3C8E82B50F141304173FAF2BDCFC164D5C45A45C4EB41247E5288354AFF417CF95288C68AEFC1500E906E8A39E0C14888D4BAC2B1D141721DB1B05D6D0242E614B5A5D2EDE8C1BBD977E7C3CCF2C136DE800E30BFE2C11FFA3CE1D1B3F1C1D6649648081CEFC1':::GEOMETRY,'010400000000000000':::GEOMETRY,'01030000800100000007000000BFCA7FDBB84EF6C1EABD558933ADFBC104480ECB9F86E9412A12CA7003BBF6416DD71AB3C8E3F0C156A03F3C8261F141C85389CBE1C9FA416441BDC7A8F4E4C11EDDB4E78736F64190D3F5202D22EF4124C55BDD2B34EA41DCC6F70EBF7DF3413A0AC9F97383FDC1001D57DF47A58B41900EB9E84621C8C171D3283A0C4A00C2205C1B3A9EBDD4C124AB60F82794F8C1BFCA7FDBB84EF6C1EABD558933ADFBC104480ECB9F86E941':::GEOMETRY], '01020000400200000010DF7E561C80D641D8D73C1E3D740142DC5AA65F0C20EB410034FB78DA16FCC1347F3DF6DFBBF641D0B9D8226A12EE41':::GEOMETRY) 
    rsg_test.go:125: Last executed (22): SELECT array_remove(ARRAY['2016-12-05 22:10:39.000242+00':::TIMESTAMPTZ,'1993-11-09 21:57:02.000418+00':::TIMESTAMPTZ,'294276-12-31 23:59:59.999999+00':::TIMESTAMPTZ], '2022-11-05 21:06:04.000238+00':::TIMESTAMPTZ) 
    rsg_test.go:125: Last executed (23): SELECT array_remove(ARRAY[e'\x0b ':::STRING,e'\x06\x0ey5-F\x04\x7fD':::STRING,'X':::STRING,e'\x03\x01\\':::STRING,e'\x03py\fWAK$_':::STRING], 'X':::STRING) 
    rsg_test.go:125: Last executed (24): SELECT array_remove(ARRAY['\x9a3c80a8a551fb':::BYTES,'\x0fe230b40d18':::BYTES,'\x524789ae95':::BYTES,'\xff8c':::BYTES], NULL::bytes) 
    rsg_test.go:125: Last executed (25): SELECT array_remove(ARRAY[4055634451:::OID,1366408462:::OID,2260659322:::OID], 68235032:::OID) 
    rsg_test.go:125: Last executed (26): SELECT array_remove(ARRAY[]:::UUID[], 'e0ada9c8-8316-43ba-a314-984866db7dd2':::UUID) 
    rsg_test.go:125: Last executed (27): SELECT array_remove(ARRAY['194.170.50.125/12':::INET,'ccf4:a741:fa45:b993:6e05:8d0:eb05:8be5/74':::INET,'32.54.226.90/11':::INET], '7e27:b9f0:103d:becc:1cbf:a06d:92b0:a935/88':::INET) 
    rsg_test.go:125: Last executed (28): SELECT array_remove(ARRAY['80000000/0':::PG_LSN,'ED957EDC/B0BF66BC':::PG_LSN], '24F06E06/EC7A27A':::PG_LSN) 
    rsg_test.go:125: Last executed (29): SELECT array_remove(ARRAY['OR7#X':::REFCURSOR,'r':::REFCURSOR], e'1c$O\x7fN1Ma':::REFCURSOR) 
    rsg_test.go:125: Last executed (30): SELECT array_remove(ARRAY['21:26:38.528272':::TIME,'08:59:09.014236':::TIME], '12:13:54.47497':::TIME) 
    rsg_test.go:125: Last executed (31): SELECT array_remove((SELECT NULL)::timetz[], '18:33:58.955594-05:51':::TIMETZ) 
    rsg_test.go:125: Last executed (32): SELECT array_remove(NULL::jsonb[], '[true, [null, 0.26110058916604534, false], 0.007767503902744055, 0.5378519793412396]':::JSONB) 
    rsg_test.go:125: Last executed (33): SELECT array_remove(ARRAY[B'00001001011100101000110101110001110010001100110000101101001010101011101'], B'111100011000111110111011') 
    rsg_test.go:125: Last executed (34): SELECT array_remove(ARRAY['naws':::LTREE], 'foo.bar.baz':::LTREE) 
    rsg_test.go:125: Last executed (35): SELECT array_remove(ARRAY[NULL]:::anyenum[], (SELECT NULL)::anyenum) 
    rsg_test.go:125: Last executed (36): SELECT array_remove(ARRAY[]:::RECORD[], (534795143:::REGPROCEDURE,)) 
    rsg_test.go:125: Last executed (37): SELECT array_replace(ARRAY[true,false], false, false) 
    rsg_test.go:125: Last executed (38): SELECT array_replace(ARRAY['BOX(-0.3096817981258907 0.788558197976213,1.008371306139147 1.165209063767811)':::BOX2D,'BOX(-0.7963501609507567 -1.9992082745039896,1.9862585521096272 0.18582353818850916)':::BOX2D,'BOX(-0.051250996312481534 -0.13161034453078896,0.8631734590883378 0.48778837565760114)':::BOX2D], 'BOX(-0.43395616551091054 0.07051714696040978,1.475415529699001 0.23018961562451395)':::BOX2D, 'BOX(-0.25889243175074483 -1.4765493448726923,0.6152100473479766 0.5204107770431351)':::BOX2D) 
    rsg_test.go:125: Last executed (39): SELECT array_replace(ARRAY[6507867229163253712:::INT8,(-8901653345879236954):::INT8,(-1871207190476431389):::INT8,(-4183192760092408946):::INT8,(-291106290821615562):::INT8], 4850897174729815451:::INT8, (-32768):::INT8) 
    rsg_test.go:125: Last executed (40): SELECT array_replace(ARRAY[]:::FLOAT8[], NULL, (-1.4277077947677135):::FLOAT8) 
    rsg_test.go:125: Last executed (41): SELECT array_replace(ARRAY[2379786.63513435002:::DECIMAL], (-7.32639652179360663E+36):::DECIMAL, 48508475840622550.42:::DECIMAL) 
    rsg_test.go:125: Last executed (42): SELECT array_replace(ARRAY['1997-01-26':::DATE,'1976-09-24':::DATE], '1990-12-21':::DATE, '1983-01-18':::DATE) 
    rsg_test.go:125: Last executed (43): SELECT array_replace(ARRAY['1982-11-01 01:56:49.000117':::TIMESTAMP,'2024-01-11 12:48:57.000053':::TIMESTAMP,'2029-11-18 17:03:08.000401':::TIMESTAMP,'2017-10-02 08:05:03.000958':::TIMESTAMP], '2023-08-16 10:28:00.00088':::TIMESTAMP, '1972-05-09 05:55:34.000591':::TIMESTAMP) 
    rsg_test.go:125: Last executed (44): SELECT array_replace(ARRAY['65 years 1 mon 725 days 04:28:35.853955':::INTERVAL,'-73 years -11 mons -649 days -08:19:12.436162':::INTERVAL,'00:00:00':::INTERVAL,'-65 years -9 mons -504 days -06:55:47.821171':::INTERVAL], NULL::interval, NULL::interval) 
    rsg_test.go:125: Last executed (45): SELECT array_replace(ARRAY['01040000C00600000001010000C0721464D48761FA4138BDB6837AD9FF415EB1A0C05439E9C180412AFE6E2CE04101010000C02054AC2FA39ED64147D80C61B5C4F0C100BD24A1A3C6DD414ADB7E6ED9ECF54101010000C050D1A48D7BFDD0C1AD62709F19CAFDC1D8FC1F249106FE412E73EE70ABEB00C201010000C0CCDFD9E116CFE3C1ED53D6EA7925F2C1809B3656D2A5F8418C3A2A249346014201010000C0113A9911D51100C2EC1884CC87A1DCC1C8584C8157D1E0413821FDFE56E2EEC101010000C0824350FE6E66F541043B84E446780142A02B3EC19ADEE5C104F60D06E1BBEA41':::GEOMETRY], '01020000800300000084970B02604201C26AA078FBD00101423A9064EADE02F1C172C1C8F4D73DFAC1AE79C4F0E9C9F241A0C397F24BCEFD4152ED2F89C070F8C154BEC7B39885E3411883C28CED63D341':::GEOMETRY, '01020000000600000030ACC2DCAEFB01C27C1DA4502A7FE1418846DB0D03D8F8C1C0242ADBE0A2E4C1A494C73207D1EDC1D037DDD1DFD101C250F509456208F141C078C4DBB127DBC15830E9264241F341F4927279ACAFE341BC0569A208B8F2416C382E63B031E741':::GEOMETRY) 
    rsg_test.go:125: Last executed (46): SELECT array_replace(ARRAY['01050000E0E61000000600000001020000C00300000038C5EAA86B5D3940D7AA000EBA0B50C0B41CB81B97C1F8415677D07F46E7F64188EF2C5AC926604080DB3A567132E0BFEDC2026DD749F7C196CF3148527A014206223D1365885040FCA4BF10CF2A434070DEE762D4CCF24169A5B8DD099DF4C101020000C0040000000065F1C958C062405075FAC63A6F4DC07775E846B3B6F4C1B052D8024052D0C1405924BBA735304080365FFB1B5F24C03E692855DFBBF14114CEDB3098A1F6417C6D15CF09F343C0B093FD88FDD74F40941B9EC62B95E4C1A00883BFBA8EDD41CA867F40B31C50C0543543D7B4323040100DA0B24CABF4C174F03874068CE04101020000C002000000BCF5CD10E5466440D2BBE054F0E63DC0CEC87EB06DC8F1C128B2858DA660F741DC8EC519A92D62402818011FFBF034406071A6E3A10501C2F8AC1B996C6AFF4101020000C003000000DE1BD4E129F060C06019C5DE58541D40645F6D51D0F6F6C1CAB8F34847A8F641B47D00BDB064584014D21C8A0FE039404A5187DA8512F341FED708CA0E3CF8411478EF2DCECB6540B4E2EF5778D64A4020799ED13160D3C160E0F7714284F9C101020000C002000000C079C58A658F4DC0FA7393E6A85B544080AEA2BEC4D7AC41EA223E1AB6C5014232C7C84999DA62C098D85F816048554072B9035728ADF741D080F53ACA86D54101020000C00200000070070ACDBDF32AC018B45B25176B2CC0904415151948DB413FAF76FC777DFFC1D4F3370A45FF4440C29810CCB4824DC0B0EF55443EF2DA41BEB69599C7A6EFC1':::GEOGRAPHY,'0104000060E6100000010000000101000040E40E04379F1A414021250D22F28A54C0B868DCD3731DE741':::GEOGRAPHY,'01070000A0E61000000100000001030000A0E610000001000000050000004C4C8D08C84261403E1642826F294EC0C09EED6A0E950242884E995A55AB374000D6FD7D9EB32040589DCF5328FED8C17832A8A92FE03AC0683116E873272A409B252E9EDF16F4C17242EFE8039263C06CDFDC177C6A4B404639E1BEB91F02424C4C8D08C84261403E1642826F294EC0C09EED6A0E950242':::GEOGRAPHY], '01070000E0E61000000500000001050000E0E61000000500000001020000C005000000D53E030E6C4462C0876CE22C12A740C04CD51088360FFD41AA4D3B9DA8300242A357FB76531864C072F15262B8764EC033396DD3C23DF2C11865619EE894DC41E8DBB107518161C00A8FBD9CCC624EC0B6000B30A009014284CD6D405EB9F24140ADE0A2333448C0655081C3626C40C07DDA78B3D54CFEC1F4FFF193D2E8FF411CA7F7EBD5BB5A40D096BCB9A7D521C058FE509132FBD6C134F831C37982F54101020000C004000000BD04AC20943965C0A8583F5D6BD951C03CADA50586E8EE41CC425E49532AF9413144F4EA430D61C083A6A66902164DC0B46FF8AAB0CEF6C1EA136E410EEEF44196022EB3E31351C09876CB2D94952DC06066ECBA0090D441407EBA1FE8E1AE41DF725AF5C1B660C000578C33CED40BC0C8EBD2444CE1DFC1D04815BC044CCDC101020000C005000000C8425EF9B31E444045AB2C6CDDCE53C07A2AEE68E3C1FFC15021A58E5B66EDC1D698210D9F52524098523048356D4BC004693B398B59F7C19896481701A9F5C128F29C6E75605440ADD09E0C6C1B40C014CA025D19E6E941003BE15F341191C1FCA94D7132EB5940AE8F13FDBAB840C0562D142545740042E0E684D9D06AE0C104BA6C46B2BE5340A0A2125396B62B40307745FB5CF5F1C1C83F33AB2380F6C101020000C0030000002CAD2C3E65575C40B88F24DD31D7464052F7E6EBB020FCC193C89719888C02C2AE69D6542BB360C0783B0C3CF98C4240CA161DAF6C89F641B6B6889DD2760042945012A90EB360C08AB0A5DA8D4842406AD1E116EDF6F5C1F9E62EB8C057FFC101020000C00300000076FD0AD09D4764C06EE9137767E753C000B817D915D351413360C0E0D0FFF8C1A0211C7F2E053DC0A74EF4202F5048C08E02C3920B03FAC138BEF02C2449FF41E850B57397F544406BBC7F11A7A64EC03C29953FA08FEC4180512058B037D14101060000E0E61000000700000001030000C0010000000B000000B2A8973F6B5158C060AC795FA5A21DC0B4B5DDF670FFE841D001207C8299EB4184681A3637965DC08A14A79D354543C040943E73C8B1BA412175EC821C45F9C1C4FB21B0F882564002AD987CBE0B52C01CDEB410327402C2AAAF41F39E1B00C2E44DBCC1BA605C408BA17E96F6B255C010B758942B10FF41809A57EE0A7AD4C1987075C75D4461408052FE408D8E54C0C1BD6185BE8701C258C7CA3FD1C0FC41C0258FFCCBE910C0A0972CAA0BD95240DE485C79B1D30142C0190D6EB977CF41302A73779AD650C0D686DE2080405540AD10C32A5BABFDC1AE08054BE1F7EAC1CBDA9CEF09E753C0C6D56C96459F524000C4B3DAED309B41073CFDE386E1F6C1CB6121BD25995AC08004727CC78E544036B736AAE6A8F0C114A4238662C3EB4188934495D7AF61C0B4D1A62A25CB4D400EF0D7ED8220014200AE7FCFDEAC7941B2A8973F6B5158C060AC795FA5A21DC0B4B5DDF670FFE841D001207C8299EB4101030000C00100000005000000D0794162FA9A63C0801931DC7F410E40000BA03476EADBC18CCE7852EB6CD9C160D7DF0970DB3FC0A0008D3E617817405035EB2AB75BE9413BD763A838A701C2D850C41DB2755F40E406456967CF33C0104B63C0B517D641CA920CADD0D5F04180B14D3E8D3752C0FCB6721B273A5340007361B694778241404D6C5B2201D541D0794162FA9A63C0801931DC7F410E40000BA03476EADBC18CCE7852EB6CD9C101030000C001000000070000007E61A1CA61A163C0B093B397D3DC28C0C0F78664DE92A5C1F476E4C26867F741845C3C660F7E634010CE5FC89E4F2CC0509055D56AECFDC11F244688082FF7C10C385AC3C0E75E40780086F6D9002E40E81605BBBCBDD4C1FBF2D060110801C2906014E56D5A2DC0780A1581343B394054389216BC4DDAC1FC0FCAB90536DDC130B2E66164895BC016492E513D175540B86F6EC71E19EB41063847C18F28ECC1AE70EA8E87225BC09073F76A4DD8474080C7B8273804A2C11017BBC2A951CC417E61A1CA61A163C0B093B397D3DC28C0C0F78664DE92A5C1F476E4C26867F74101030000C00100000007000000B9055F1141CF57C070F4E7643DC323C0350DF47790CFF0C100921D1DD8CC934136035F6D7CEA51C0373D1826656654C0A6A88629FD1CFAC189BC5D9B9EB000C25AD73A569309654070DE6A38A57034C08EB91A2406C2F4C17EBF1C5EEF57FEC1E4FA2DCFA4D34C40D0FAB9EF9A023D4090608328ADECD4C15055173384A7FD41ECA2073DC9C15440FCF5B534BEBA5340231ED1F19F81FBC19CA10577A7FEE941B403A9B2943542C0004A1B543D7B2B40400D7AC66C5DBFC17E0DE3E37807EFC1B9055F1141CF57C070F4E7643DC323C0350DF47790CFF0C100921D1DD8CC934101030000C0010000000B0000005D77A45D09A465C01C17CBDF577943C06AA866728748F141C4C420AF6E88EAC182D87710C21260C0F89A31A4684A53C070518A6074A5E141ECDDA16A612EE5C1F0026997A2E721C01C4D8DD5094150C09C1122EAA0F7F941DCE3099DA425E841E8949ABBE8C25540AAFC03C6D95856C0AF69C8A4E59202C200E2FBC581DDFD41E003864EE3A04F40A42BB6FAF22C44C0E6D3FFF18AE401C2A8DB162C3CE4D2C158FDBE35AC8153C04053F4DC4EF504C0F5C5830DA0E5F1C1D8DD13E0EFD8D941AA8B95AC7E195FC0F0BEFDC5FA1445408A4902FC3242F341987822B37883F1C1AACD00A184395FC0E040F09BBBB51C40A4CC0B5E9A99E241C810BD7D825EEA413D0F46EA72D460C0A0850E9F24CB094088F40569C1C6F641EA6BFAFC37B1E0C1127F6B02575062C0A80F01B8669024C02EE17F940D5901C2B88CB4FA47ECFB415D77A45D09A465C01C17CBDF577943C06AA866728748F141C4C420AF6E88EAC101030000C0010000000D000000A0B1396835BB27C0C0FE4B7A459F0EC068464BA0C14DE241A89CA64D837CE6410070FAF45D0129400276C6411BED43C08655FEB8958BF7C1ECAD33379E88FBC1F02AC32B5FD3414056B8F0C9E5B14BC018D4839E542FFB419CF2539385BCEEC12413CF2FAE5F434008A89A2E4DA22EC0A0D981FD5FA8E8C1CC1CE6150B7EEF41547AAA427ACB444020A9FD4E979414C012A9035B4143E9C1D3235084B94C02C288CF258BF74A6540234E2CEE954A45C080F78773795FE6C130400F2F8C91E941A8FE4B07715B6040746C398AD9DA35C0A6136D4F8B7DF3416887416F92AED4C1986643AEBEB35740B4944C5E8B565640ACD8FFC51773DFC1743BD0E70B1802C244C068D0BA7041C0B4A46222CF424D40F062A3213563C1C1D75CCE16CEB7F3C1B4435EA146A456C064B2F87F8F51484003DE55A4C77500C28CA76E93E69CFC41BE4E9CF0E34660C06CD0A980140A3940E474EDED703B004217F2D9F14444F2C1BBA80A65546952C04074AB6C13F52540D0FE948A84FA01C23A4F00955EEFF341A0B1396835BB27C0C0FE4B7A459F0EC068464BA0C14DE241A89CA64D837CE64101030000C00100000005000000A43B88F9255B60C0177D8C59714B4AC02CEC6F108A2CE14108481A8CD1EAEB419EB767410C2C5BC06B75E10FFCC551C04080EF7051ABFB41DAF4B6E23B93EDC104CCF44F730D4340385B037BF22D33C038EC9D99CD12F94136662B61148001C2906988CA233E46C054E40B32A64A5140D4ADD5F8381AE6C15C8D8E06BCA4F841A43B88F9255B60C0177D8C59714B4AC02CEC6F108A2CE14108481A8CD1EAEB4101060000E0E61000000200000001030000C001000000080000002ACA7A5977E25DC02229D843C28542C02C7DF25A31ADF14116E507C77CC001C24146634543255BC073EDCFF5371546C0682EAB2E1CBFF8C1E637D93D2674F341F5C9F9461AFC50C01B99C94C44D24BC01403B5731C88E8410073D789F9EBFD411AAE0FB574476240B71C4908361656C032617211FAE10142E80E59CDB7EADD41F0855BA8FBF157405250ABB27DB64AC0A242FB71FB6F01C27E1DC5EF7BB0F7412886EB5AA37C5540801D599A5461334032348E99EF3AE7C1AE6B321B3FE6EBC1518A2623A26555C004C5E70156563B4060DEB919F673C441C07CCF2BEF8BE1C12ACA7A5977E25DC02229D843C28542C02C7DF25A31ADF14116E507C77CC001C201030000C001000000080000001CDDDBFF9AA75AC0451F968770DF4CC014532CC999D600429EAA58EC4427F641E4492A59B96E5B40F068C02195D449C0183B4E9AA981E041682B2EE796C2004282A8AD19265F6040F0B02AFFD66C3CC0F84272B115C7E741D0D3D2C9DCA5F541A48F704FFD304440E0B015EC1D671C408E9C388D876901426D5576AE185300C2E4A8583FDDDA5540144EADCD13E53B40E86BCB6D4C2FFC412C260CEF42DBFC41FD335BAB6E7C63C0CED8B8C42A53564062806B31140AE8C19A28E290D859F441CE9AE4E61E485DC0C4DE19B090564840B0A068B378E4F0C140861B8E156CEC411CDDDBFF9AA75AC0451F968770DF4CC014532CC999D600429EAA58EC4427F64101030000E0E610000001000000060000002835C4FA30A73640B8E033C7A4133CC06C0122B1AE87F441706C02E3B695C2C1289024098F7D57409C241206D59A4DC03E1BF934DE62ECC10848CA834CC2E6C12888C289C6075B40B7C414FD7E2A48C05E16D38FE169EEC1A8592325A48EFC4198B86568FA3C65402A3193D520705340F6853C82B7F2FFC138BEEB2A3099D4410C3642086C3A4040505C2D33369D2240BAA6AE9AE3B2FDC180EB01782485D5412835C4FA30A73640B8E033C7A4133CC06C0122B1AE87F441706C02E3B695C2C101010000E0E6100000D4CCAD8A460F5B404027192A39DA40C0289A1126FFF301C2B4BA90E29CBA00C2':::GEOGRAPHY, '01030000A0E610000000000000':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (47): SELECT array_replace(ARRAY[e'+6xp!\x0b':::STRING,e' |.Q\x17\x11':::STRING,'':::STRING,'"':::STRING], '':::STRING, (SELECT NULL)::string) 
    rsg_test.go:125: Last executed (48): SELECT array_replace(ARRAY[]:::BYTES[], '\x':::BYTES, '\xb3':::BYTES) 
    rsg_test.go:125: Last executed (49): SELECT array_replace(ARRAY['1976-06-08 11:03:10.000054+00':::TIMESTAMPTZ], NULL, '1992-06-22 15:44:21.00045+00':::TIMESTAMPTZ) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions3314116234
--- FAIL: TestRandomSyntaxFunctions (301.84s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-57423

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20815838?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20815838?buildTab=artifacts#/) on master @ [a76eb5de5d3fdeb4f1276eefe5fdb11e6ebe53a4](https://github.com/cockroachdb/cockroach/commits/a76eb5de5d3fdeb4f1276eefe5fdb11e6ebe53a4):


```
    rsg_test.go:125: Last executed (4): SELECT enum_last(NULL) 
    rsg_test.go:125: Last executed (5): SELECT enum_out(NULL) 
    rsg_test.go:125: Last executed (6): SELECT enum_range(NULL) 
    rsg_test.go:125: Last executed (7): SELECT enum_range(NULL, (SELECT NULL)::anyenum) 
    rsg_test.go:125: Last executed (8): SELECT enum_recv('':::VOID) 
    rsg_test.go:125: Last executed (9): SELECT enum_send(NULL) 
    rsg_test.go:125: Last executed (10): SELECT every(false) 
    rsg_test.go:125: Last executed (11): SELECT exp((-0.07261368378758704):::FLOAT8) 
    rsg_test.go:125: Last executed (12): SELECT exp(NULL) 
    rsg_test.go:125: Last executed (13): SELECT experimental_follower_read_timestamp() 
    rsg_test.go:125: Last executed (14): SELECT experimental_strftime(NULL::timestamptz, 'n':::STRING) 
    rsg_test.go:125: Last executed (15): SELECT experimental_strftime('3000-01-01 00:00:00':::TIMESTAMP, '':::STRING) 
    rsg_test.go:125: Last executed (16): SELECT experimental_strftime('1982-01-21':::DATE, e'\x14"I':::STRING) 
    rsg_test.go:125: Last executed (17): SELECT experimental_strptime(e'\x1f\x17\x02*.p':::STRING, 'Atw)7nV':::STRING) 
    rsg_test.go:125: Last executed (18): SELECT experimental_uuid_v4() 
    rsg_test.go:125: Last executed (19): SELECT extract('':::STRING, '1990-12-27 00:33:19.000055':::TIMESTAMP) 
    rsg_test.go:125: Last executed (20): SELECT extract(NULL::string, '77 years 11 mons 890 days 13:31:41.569691':::INTERVAL) 
    rsg_test.go:125: Last executed (21): SELECT extract(e'6R\x1a':::STRING, '1992-04-13':::DATE) 
    rsg_test.go:125: Last executed (22): SELECT extract('':::STRING, '1990-04-21 05:08:16.000533+00':::TIMESTAMPTZ) 
    rsg_test.go:125: Last executed (23): SELECT extract(e'\x1ee0[F':::STRING, '05:01:20.952073':::TIME) 
    rsg_test.go:125: Last executed (24): SELECT extract('r:':::STRING, NULL::timetz) 
    rsg_test.go:125: Last executed (25): SELECT extract_duration(e'\f':::STRING, '16 years 6 mons 283 days 03:41:13.488247':::INTERVAL) 
    rsg_test.go:125: Last executed (26): SELECT family('ad77:655:92c:ee21:2987:c9f1:b582:cddb/85':::INET) 
    rsg_test.go:125: Last executed (27): SELECT first_value(NULL::bool) 
    rsg_test.go:125: Last executed (28): SELECT first_value(NULL::box2d) 
    rsg_test.go:125: Last executed (29): SELECT first_value((-7889413382935499840):::INT8) 
    rsg_test.go:125: Last executed (30): SELECT first_value(1.0325427050066773:::FLOAT8) 
    rsg_test.go:125: Last executed (31): SELECT first_value((SELECT NULL)::decimal) 
    rsg_test.go:125: Last executed (32): SELECT first_value('1976-11-15':::DATE) 
    rsg_test.go:125: Last executed (33): SELECT first_value('1979-11-06 05:12:09.000472':::TIMESTAMP) 
    rsg_test.go:125: Last executed (34): SELECT first_value('1 mon 1 day':::INTERVAL) 
    rsg_test.go:125: Last executed (35): SELECT first_value('01060000E0E61000000300000001030000C00100000004000000524A91C73FA84BC0285C1DF083DC54C0DC6D691EE496FCC1C6E0DE4FBA89F6C15EC5C0C57C5862401A757A3496C44FC0F8248C6DD10CD8C1409B39710B12CDC10491808DA2935B40C4F48CA70090514080BFCD0C2521C84170737216EB53D641524A91C73FA84BC0285C1DF083DC54C0DC6D691EE496FCC1C6E0DE4FBA89F6C101030000C0010000000A000000EC9270D3F7CC51C0DBEDADB439D451C0AA56F52591A4F9C1BCFE1F5F2383E141D8982BE5C04A4C40C53F83B4D5BF51C0D0CF67659F9FD4413ED7E208796EF8C1BE30F540C551624031921CECB7B550C05C45AD7DE9AAF041D0FA9A60C842DD4188E440BC99465A4000A2DF40485744C0152E5C13B949F6C1084B2334352BD9418631CBBACE706340E2C17EF70FFE43C0BA0FDBA5771102C2DA07E8CF9C46EFC1B08B100AFCED48407417ADD4A40741407C11A50A2765E841D68EA792A476024234C8EF756D694340003093E72CB6DF3F1E8134D2570FF84164F66F65149FE0C19495B06BDD7140C050E693D06DA51E400C6ED1EAA90A02C26356D548D6D9F2C1CFF0A8C2A2BC60C0E085299A07250AC08EE4DA8897FDFCC1C8D0B8FB93FADF41EC9270D3F7CC51C0DBEDADB439D451C0AA56F52591A4F9C1BCFE1F5F2383E14101030000C0010000000700000018F65F614E4237C03C057B25F51151C0348B5B972A3CF441CC9E5A831375E74100E39791FE2F0340D8D21B2FD1724AC0728E6E74721AFCC1DEC0E5337B6CF441705C6D6362045B40209B9A24762228C024F54B9AC218F941E0BFADBF2335CA41B4B84F5E0DCD4140E00F623924431CC020D268228C80EA4106347ADC7FC8FEC10022CAB09901F33FE0CB8A7265D92840262EE43E91A3F84123ABCD03C3B7F8C11F1ED85329DF54C02CE694F1191446403EDCA5F6CAE8F4C146D58D1824CAFCC118F65F614E4237C03C057B25F51151C0348B5B972A3CF441CC9E5A831375E741':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (36): SELECT first_value('010200004000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (37): SELECT first_value(e'\x13K~\x7f':::STRING) 
    rsg_test.go:125: Last executed (38): SELECT first_value('\x':::BYTES) 
    rsg_test.go:125: Last executed (39): SELECT first_value('2016-09-09 04:18:19.000894+00':::TIMESTAMPTZ) 
    rsg_test.go:125: Last executed (40): SELECT first_value(3586437737:::OID) 
    rsg_test.go:125: Last executed (41): SELECT first_value('f1e46b15-dba5-4fc3-8552-c5d133339e28':::UUID) 
    rsg_test.go:125: Last executed (42): SELECT first_value('157d:385f:206c:1a40:d382:ff03:5f60:a1c9/87':::INET) 
    rsg_test.go:125: Last executed (43): SELECT first_value('8F23BC17/9E10ED30':::PG_LSN) 
    rsg_test.go:125: Last executed (44): SELECT first_value(NULL::refcursor) 
    rsg_test.go:125: Last executed (45): SELECT first_value('05:02:39.235954':::TIME) 
    rsg_test.go:125: Last executed (46): SELECT first_value('00:00:00+15:59':::TIMETZ) 
    rsg_test.go:125: Last executed (47): SELECT first_value('{"3Vwa1O8": {"JMw8fCza": false, "baz": true}, "b": true, "foo": [false]}':::JSONB) 
    rsg_test.go:125: Last executed (48): SELECT first_value((SELECT NULL)::varbit) 
    rsg_test.go:125: Last executed (49): SELECT first_value('ysDL.g.bnT08AzPD.z_Cy.g0JSoRZ_C.7OvU.hvZEbJ3':::LTREE) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2495000545
--- FAIL: TestRandomSyntaxFunctions (301.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20824079?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20824079?buildTab=artifacts#/) on master @ [d36cdb971710d69d43c9f242d45019ddea71049b](https://github.com/cockroachdb/cockroach/commits/d36cdb971710d69d43c9f242d45019ddea71049b):


```
    rsg_test.go:125: Last executed (4): SELECT parse_timestamp(e'C\x1c# x\x10':::STRING) 
    rsg_test.go:125: Last executed (5): SELECT parse_timestamp(e'\x10VX\x15J':::STRING, 'B,':::STRING) 
    rsg_test.go:125: Last executed (6): SELECT parse_timetz(e'\x10':::STRING) 
    rsg_test.go:125: Last executed (7): SELECT parse_timetz((SELECT NULL)::string, e'\x0bK^|7':::STRING) 
    rsg_test.go:125: Last executed (8): SELECT percentile_cont(0.6156945087617288:::FLOAT8) 
    rsg_test.go:125: Last executed (9): SELECT percentile_cont((-0.4664695454190645):::FLOAT8) 
    rsg_test.go:125: Last executed (10): SELECT percent_rank() 
    rsg_test.go:125: Last executed (11): SELECT percentile_cont(ARRAY[1.252012395325895:::FLOAT8]) 
    rsg_test.go:125: Last executed (12): SELECT percentile_cont(ARRAY[]:::FLOAT8[]) 
    rsg_test.go:125: Last executed (13): SELECT percentile_disc((-0.6254959662317523):::FLOAT8) 
    rsg_test.go:125: Last executed (14): SELECT percentile_disc(ARRAY[0.08222983163023612:::FLOAT8,(-0.46135912061676987):::FLOAT8,0.287796160512656:::FLOAT8]) 
    rsg_test.go:125: Last executed (15): SELECT pg_advisory_unlock((-1689543281):::INT8, 221820656:::INT8) 
    rsg_test.go:125: Last executed (16): SELECT pg_advisory_unlock((-201459990586749370):::INT8) 
    rsg_test.go:125: Last executed (17): SELECT pg_advisory_unlock_all() 
    rsg_test.go:125: Last executed (18): SELECT pg_advisory_unlock_shared(4705793272297086007:::INT8) 
    rsg_test.go:125: Last executed (19): SELECT pg_advisory_unlock_shared((-1083503109):::INT8, (-1086672677):::INT8) 
    rsg_test.go:125: Last executed (20): SELECT pg_client_encoding() 
    rsg_test.go:125: Last executed (21): SELECT pg_backend_pid() 
    rsg_test.go:125: Last executed (22): SELECT pg_blocking_pids() 
    rsg_test.go:125: Last executed (23): SELECT pg_collation_for(ARRAY['v':::STRING]) 
    rsg_test.go:125: Last executed (24): SELECT pg_column_is_updatable(996482932:::OID, (-13824):::INT8, true) 
    rsg_test.go:125: Last executed (25): SELECT pg_column_size('[1.5614091,-0.55706984,0.820337,1.6639581,1.6891446,0.28007358,0.9571445,0.6119995,-1.369879,1.1234027,0.8204358,-0.5046912,-0.35977262,0.011748981,0.5527054,0.2409796,0.835572,-0.98902947,0.7389062,0.27938333,1.6090605,-0.25194827,1.6991831,-0.41066375,0.5146722,0.89423686,-1.3705682,-0.27897412,1.6516885,-1.159664,-0.6712762,-0.2761645,-0.350691,2.5183594,-0.30182895,-0.62127525,1.7250795,-0.57697344,0.92442256,0.47325423,-1.1112275,0.39031857,1.0165335,-0.6338037,0.7357195,1.0963943,0.85806066,-0.85031545,0.5409964,0.29622152,-0.6280517,-0.54852533,2.418972,-0.94372034,0.18694592,0.47989887,0.648684,-0.12129194,-1.4232044,1.65075,-0.104201525,0.55584157,1.4078296,0.63325053,1.6359136,0.84169304,1.5016723,0.62304664,-1.310995,-1.5948634,0.33394727,-1.0338748,-1.180729,1.1986463,0.040570747,2.0865226,-0.56052125,1.341444,-2.5350912,-0.37827155,0.20877706,1.2327799,-0.07946888,0.16868483,0.2999083,0.44599262,-0.37150985,-2.4835486,1.7023835,0.50788337,0.64784974,2.5860562,0.79454195,0.061223015,0.58042,-1.3538415,-0.7487064,-0.02388222,-1.4437249,-0.97097546,-0.17646788,0.64994305,-2.1580288,-0.729282,0.80643725,0.5476524,-1.3147634,-1.4289769,-0.2873867,0.22184837,-0.45299935,1.7233838,-0.6184436,-0.031083168,0.09877752,0.8526264,1.4824275,-0.9852759,-0.5505542,-0.692477,0.60079503,0.8663901,0.9979615,-0.57277757,-1.1180497,0.26804936,0.82709265,2.4388242,0.6138132,0.04110973,0.46087256,0.42209473,-1.832403,0.7018557,0.60805553,0.78116035,0.9090679,0.25201416,-2.0192554,-1.6617787,-0.8418854,-0.47446707,0.69333184,0.21138269,-1.0696728,0.58894634,0.13437322,-0.7777745,1.2302392,0.3247341,0.36974803,0.05008591,0.5296886,-0.96518964,-0.04488463,-0.3742087,0.843125,-0.8277927,-2.3078108,-0.45059484,0.64934844,-0.04748355,-0.53575546,-1.3979839,1.3856552,-0.528585,1.8399919,0.854489,-1.4236386,-1.1103777,-0.22330083,-1.178947,0.12169399,1.9520339,-0.36446267,1.196898,-0.264758,0.6270832,2.0132415,1.4755462,-1.5858614,0.08583829,-0.047566112,0.32126227,0.12701494,1.3310682,-0.7950105,-0.09904193,0.23340331,-0.24188897,-0.35071972,-0.3630067,-0.486481,-1.033397,-1.2869929,0.08393579,0.22725749,-0.3401771,-0.90945596,-0.96561813,-2.004411,-0.15805097,1.6422784,-0.22541757,0.9020635,0.44663703,-0.6766567,-0.70959646,0.6306195,1.1603553,0.84220743,0.6551524,1.419131,0.46846396,0.7755461,0.15976438,0.9644237,-0.17155176,-0.9388016,0.8953372,-0.3659691,2.2624447,0.36568645,1.3462057,-0.2508777,-1.1742522,0.75197965,0.38504547,1.7167135,-0.4560072,-0.40661567,-0.58619803,-0.2329504,2.6056094,-1.3264596,2.0385242,-1.9275297,0.9967978,-1.0748662,-0.79566073,0.53181666,-1.4968718,-0.23266415,0.7189041,-0.3395733,0.123506576,-0.74891096,0.5360712,0.4942462,-0.6886806,-1.1775709,0.20609449,-0.79981285,-0.052254975,-1.2990819,-1.336823,0.030920438,-2.5473976,-1.5155553,0.8580058,-0.9487207,-0.4223092,0.19168283,1.7299361,0.0017519258,0.47101602,0.11081538,-0.9043603,-0.5770921,-1.7380521,-1.3600236,0.16747758,0.677153,0.752133,1.0012481,1.5195186,1.9488206,-0.039720826,-2.5769753,-0.47828996,-1.9325023,1.8136328,-0.110162914,-0.61824584,0.50599605,-0.8316866,-0.24856144,-2.6526155,0.020521216,-0.78112,0.26298255,-2.1606286,0.5315103,1.8961034,0.3931308,0.32441378,0.5030794,1.2171293,0.061078567,-0.9712616,-0.8448098,-0.36844158,0.7992256,0.98630446,0.5605019,0.20055293,0.53221714,1.5590724,-1.2120358,-0.092289664,-0.15545465,0.7819493,1.7978026,0.38072097,-0.28506717,-0.3506381,1.1419978,1.5812607,0.87645,0.325887,-1.4362528,-0.74778736,-0.7639274,1.6571068,-0.31830662,-1.1050458,-0.9008867,0.5102013,0.1387255,1.2488341,0.51828414,-0.74155194,1.8185954,0.74423116,-0.80457,1.2993858,-0.30190516,-1.5864804,-1.6815329,0.48554373,-1.0337385,1.2084018,-0.028847916,-0.8403354,-1.0562707,0.22199696,-0.6991934,-1.1654783,0.9355617,0.3028111,0.5023757,0.6227298,-1.0855689,0.2616845,1.3599243,-1.1192344,0.92415375,-2.3637784,-2.2272453,0.90181345,-0.76174635,0.10942115,0.6269045,-1.032489,0.12504901,0.1751071,-0.48553148,1.026454,-0.19105262,0.21930414,0.7385603,-0.30176517,1.0251238,-0.8623741,-0.025438739,0.46624956,-1.6123625,-0.19751917,-0.4691929,0.2980059,-1.2679751,0.025987376,0.47193232,1.7432747,1.0791056,-0.08624988,-0.24434398,-0.3945495,-1.2749205,-1.4038888,-0.9611303,-0.85812706,-0.38018078,0.28130087,0.19766524,0.31317598,-0.45429865,0.97972816,1.1662881,0.6006768,0.9559364,-1.145505,0.6228816,-1.1692044,0.7035634,1.5472856,0.7147107,-2.0610294,-0.68701595,-0.8438743,1.0808103,0.36575145,1.9798583,-0.108623855,-0.5293845,0.3205703,-0.71954983,0.038213752,-0.42259833,0.31936756,-1.011363,-0.8038275,0.2991433,0.29100007,0.8051444,-0.99403787,0.708224,0.12518291,-0.30004707,0.10097902,0.78613406,0.09767752,-0.39074835,0.369378,0.25882748,-0.7422779,1.3243934,-1.2546966,-0.41222358,0.4710851,-0.8357328,-0.9858012,2.090965,0.33643314,-0.048793543,0.90907824,1.0152668,-1.5516224,0.17613979,-0.85753715,-0.9221457,-0.23296852,1.0960517,0.8245344,-1.7148191,0.37178048,0.64070714,1.6179959,-1.5155705,0.94041234,-0.27957276,-0.13493328,1.1306216,-1.1479064,-1.1963824,-0.891641,1.7369404,-1.7551208,2.17383,0.2530876,0.5841191,1.3577383,-1.2585962,-1.488762,1.5477214,1.1050589,-0.19928204,-0.22900385,-0.87072176,-0.7295646,0.20188431,0.41271836,1.0985199,0.587477,0.6143063,-1.481084,-0.3091363,0.7735285,-1.1466167,0.16029114,0.6626821,0.3569337,0.104679555,1.4441313,-0.52161735,-0.4435682,-0.35592407,-0.5025134,1.5522304,0.8494703,0.9144449,-1.0544674,-1.278427,1.0364068,0.266767,-0.13373664,-0.82194483,-0.9661373,-0.37434375,-0.7157838,0.6410838,-1.123235,0.3298617,1.0275661,-1.440496,-0.86821485,0.4024587,0.27823833,-0.3231928,-1.0226934,-0.55558133,-0.3465606,-0.272205,-0.4298631,1.6748135,-1.2446918,-2.992291,0.4115011,-0.35558215,0.82563764,0.15523165,0.367294,-0.6381044,-0.5550035,0.019683104,0.76405823,0.011143347,-0.039147735,-1.1543024,-0.43205395,-0.7271496,-0.15126362,0.0245946,1.8941102,0.7480452,-2.4790585,-0.030768786,-0.7837418,-0.5648228,-0.8555236,-0.07099085,-0.17601141,1.3817203,1.8875785,-2.4720004,1.9394643,0.40492663,-0.2696227,0.18731438,-0.23027958,-0.89305234,0.2482541,-1.0572033,-0.10736202,-0.24605921,-2.494805,-1.1478748,0.73592955,-0.0044775107,-2.142576,0.84427017,-0.80431116,-0.7844143,-2.4677525,1.0465912,-0.743013,0.62710077,0.5299638,-0.660304,1.4274424,0.26220316,0.47612184,-0.5814362,0.6052435,-1.1200342,-0.12710275,1.4727505,-1.2471116,0.06536266,1.3382609,-0.58315396,0.11849469,-0.031939235,0.018512832,-0.62187696,0.280801,0.65921205,-0.94095206,-0.2684128,1.4429151,1.9708905,0.66486853,-0.05150363,-0.9239418,0.4196071,0.18263091,-0.912245,-1.356465,0.44777447,-0.6551547,-0.19207136,0.03302071,0.14265728,-0.40277585,-1.7562977,0.028212905,-0.36891845,-0.10803639,0.18347354,-0.5633089,-0.5344417,-0.5400211,0.56935465,1.9843286,-0.32244185,0.40311307,-0.6008992,1.1710249,-2.16933,1.4314989,1.2390705,0.57666373,-0.09601734,-0.44130695,0.12722063,0.124479786,0.50519854,0.73144764,1.1998658,1.4291512,1.6751214,-0.7069769,-0.5007533,0.84678817,-0.06768456,-1.2320035,-1.1923412,-0.8259973,-0.20263748,-0.46126723,1.1064037,0.41617796,1.1930056,-0.1243641,0.044454865,-0.56890213,0.75211906,-0.583258,-0.7897985,-0.33998248,0.3321214,0.10306259,-2.5579638,-0.43498266,-1.1480745,-0.76311594,0.9615935,-1.026445,-0.43727162,-0.84755856,-0.77473325,0.17352527,-1.1074475,1.1094595,0.3920594,-1.3128251,-0.13804634,0.8217563,1.1470015,-0.46370012,1.7577258,0.2996033,1.635334,-1.078245,0.066479385,-0.28571394,-0.34705067,0.2310585,0.6948993,-0.38452289,-0.41280872,0.16011693,-1.0051428,-0.024821445,0.5390188,0.43162718,-1.2757939,-0.49053097,-1.4765277,-0.7325225,-0.76868534,0.90998137,-2.422459,-1.0446095,0.34192392,0.17577568,0.45422238,0.9074448,1.567007,2.083993,1.0886211,1.3506501,-0.47425672,0.038488913,0.24171379,1.4074157,-0.14895132,-0.37129596,-0.5740038,1.8606297,-2.1753306,-1.052099,-1.4401963,-2.1551929,0.44126418,0.6293508,-0.76920176,-0.06274038,-0.22138274,-1.8467599,0.49610087,0.49313304,-0.27569857,0.11992894,-1.2015779,-0.62069523,0.91174614,-1.2548608,1.6223372,-1.490295,0.108403064,-0.43582764,-1.058276,0.33623716,-1.0763228,0.6724105,1.1345336,0.11228251,0.19065756,-1.6521252,0.036916357,-1.151305,0.37270004,1.5813072,-0.38134938,-1.1942176,-0.2155687,-0.4715112,1.2868534,-0.020547595,1.975446,0.005970468,-0.7413538,-0.6598101,-0.42796424,-0.05951738,0.37620717,2.4709117,-0.72520816,0.2401947,-2.5467236,0.5449531,0.44240728,-0.45864952,-1.0746129,0.5847746,0.33593923,-1.0235204,0.06965138,0.33679554,1.04351,0.6313474,-0.4624957,1.5175236,-1.6094536,-1.4217316,-2.012107,-0.7397544,0.8750201,-1.0252115,0.29159752,-0.4789185,1.9842921,-0.9427354]':::VECTOR, '2001-01-01 00:00:00 BC':::TIMESTAMP, NULL, '33 years 8 mons 134 days 07:23:10.917605':::INTERVAL) 
    rsg_test.go:125: Last executed (26): SELECT pg_encoding_to_char(8483109507903373134:::INT8) 
    rsg_test.go:125: Last executed (27): SELECT pg_encoding_max_length(4303124623685297478:::INT8) 
    rsg_test.go:125: Last executed (28): SELECT pg_function_is_visible(3183255013:::OID) 
    rsg_test.go:125: Last executed (29): SELECT pg_get_constraintdef(3902770855:::OID, false) 
    rsg_test.go:125: Last executed (30): SELECT pg_get_constraintdef((SELECT NULL)::oid) 
    rsg_test.go:125: Last executed (31): SELECT pg_get_expr('Q0enA':::STRING, 4174973575:::OID) 
    rsg_test.go:125: Last executed (32): SELECT pg_get_expr(e'\bqQ\x02z':::STRING, 3927659690:::OID, (SELECT NULL)::bool) 
    rsg_test.go:125: Last executed (33): SELECT pg_get_function_arg_default(1944955115:::OID, NULL::int4) 
    rsg_test.go:125: Last executed (34): SELECT pg_get_function_arguments(NULL) 
    rsg_test.go:125: Last executed (35): SELECT pg_get_function_identity_arguments(0:::OID) 
    rsg_test.go:125: Last executed (36): SELECT pg_get_function_result(0:::OID) 
    rsg_test.go:125: Last executed (37): SELECT pg_get_functiondef(58035992:::OID) 
    rsg_test.go:125: Last executed (38): SELECT pg_get_indexdef(NULL::oid) 
    rsg_test.go:125: Last executed (39): SELECT pg_get_indexdef(2807275348:::OID, 6133567838573083821:::INT8, true) 
    rsg_test.go:125: Last executed (40): SELECT pg_get_keywords() 
    rsg_test.go:125: Last executed (41): SELECT pg_get_partkeydef(2950565761:::OID) 
    rsg_test.go:125: Last executed (42): SELECT pg_get_serial_sequence('gC':::STRING, ']':::STRING) 
    rsg_test.go:125: Last executed (43): SELECT pg_get_userbyid(3346354327:::OID) 
    rsg_test.go:125: Last executed (44): SELECT pg_get_viewdef(3739306285:::OID) 
    rsg_test.go:125: Last executed (45): SELECT pg_get_viewdef(4279756833:::OID, false) 
    rsg_test.go:125: Last executed (46): SELECT pg_has_role(e'0;\r\x04\t':::STRING, NULL) 
    rsg_test.go:125: Last executed (47): SELECT pg_has_role(986847131:::OID, e'\x17\t\x1fXtL6':::STRING) 
    rsg_test.go:125: Last executed (48): SELECT pg_has_role(NULL::string, '':::STRING, '7QR':::STRING) 
    rsg_test.go:125: Last executed (49): SELECT pg_has_role(e'\nyL,c{;N':::STRING, 2202146471:::OID, (SELECT NULL)::string) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2491073998
--- FAIL: TestRandomSyntaxFunctions (302.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20838391?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20838391?buildTab=artifacts#/) on master @ [011de4912a978d4aec084efa687ae8b7e9484b3f](https://github.com/cockroachdb/cockroach/commits/011de4912a978d4aec084efa687ae8b7e9484b3f):


```
    rsg_test.go:125: Last executed (4): SELECT inet_same_family('70.238.249.5/2':::INET, 'a295:4b42:acff:7e03:41fc:7050:61d8:5215/64':::INET) 
    rsg_test.go:125: Last executed (5): SELECT inet_server_addr() 
    rsg_test.go:125: Last executed (6): SELECT inet_contains_or_equals('ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff':::INET, '70.169.207.133/10':::INET) 
    rsg_test.go:125: Last executed (7): SELECT inet_server_port() 
    rsg_test.go:125: Last executed (8): SELECT inetin((SELECT NULL)::anyelement) 
    rsg_test.go:125: Last executed (9): SELECT inetsend((SELECT NULL)::inet) 
    rsg_test.go:125: Last executed (10): SELECT inetout(NULL) 
    rsg_test.go:125: Last executed (11): SELECT inetrecv((-20660):::INT8) 
    rsg_test.go:125: Last executed (12): SELECT information_schema._pg_char_max_length(2148551875:::OID, (SELECT NULL)::int4) 
    rsg_test.go:125: Last executed (13): SELECT information_schema._pg_datetime_precision((SELECT NULL)::oid, (-632310443):::INT8) 
    rsg_test.go:125: Last executed (14): SELECT information_schema._pg_char_octet_length(1707016335:::OID, (-545795890):::INT8) 
    rsg_test.go:125: Last executed (15): SELECT information_schema._pg_expandarray(ARRAY['K':::REFCURSOR,e'u\x0fBwK\x1fC"e':::REFCURSOR,e'\x00':::REFCURSOR,'':::REFCURSOR,'':::REFCURSOR]) 
    rsg_test.go:125: Last executed (16): SELECT information_schema._pg_interval_type(NULL, (-447315113):::INT8) 
    rsg_test.go:125: Last executed (17): SELECT information_schema._pg_index_position(3632008730:::OID, 5970:::INT8) 
    rsg_test.go:125: Last executed (18): SELECT information_schema._pg_numeric_precision(2651083449:::OID, 905711824:::INT8) 
    rsg_test.go:125: Last executed (19): SELECT information_schema._pg_numeric_precision_radix(1839950547:::OID, 813093020:::INT8) 
    rsg_test.go:125: Last executed (20): SELECT information_schema._pg_numeric_scale(2753048539:::OID, (-814141385):::INT8) 
    rsg_test.go:125: Last executed (21): SELECT information_schema._pg_truetypid(NULL, (e'\U0000830B\U0000C0BB\u04F2\U0002C056\U0001F1F7\U00004B55' COLLATE de_DE,)) 
    rsg_test.go:125: Last executed (22): SELECT information_schema._pg_truetypmod(('e1a14dda-fdf1-444b-b5f1-c257562ddc64':::UUID,), (B'101011000011111111110111001100110100110110',)) 
    rsg_test.go:125: Last executed (23): SELECT initcap(e'%\t=N':::STRING) 
    rsg_test.go:125: Last executed (24): SELECT int2in(ARRAY[(-1728):::INT8,476:::INT8,(-58):::INT8,(-23917):::INT8]) 
    rsg_test.go:125: Last executed (25): SELECT inner_product('[-0.7422183,1.1694207,0.84334123,1.61072,0.2776324,0.5843048,1.2357769,1.421742,0.7693224,0.8183571,-0.11722624,-1.0030463,-0.17333996,-2.1990871,0.08400465,0.107868,0.14467172,0.25307155,0.011478208,-0.26117212,-0.31217137,0.9562907,-0.4430329,-1.7262461,-0.2967103,1.8527838,-0.52169853,-0.092912816,-0.75368035,-0.67214984,-0.5914266,0.7326039,-1.3810489,0.952805,-3.3638015,-2.4116218,1.024288,0.78896874,-1.3267136,-0.46868384,1.6629999,1.6594288,-1.5215582,1.8944738,0.037370227,0.3026367,-0.27265725,-0.3078512,-1.5484388,-0.5742516,-0.6211987,-1.3468933,0.86677057,0.588964,-0.5313869,0.8873227,0.057026006,-0.4435209,0.5426003,0.27666375,-0.4319813,0.3313082,-0.7445322,0.28830725,-1.471886,0.004362366,1.5557047,-0.19555254,0.1503765,1.0743856,1.2066494,-0.4409092,-0.52779025,0.45305794,-0.29538116,-0.005607857,1.0210131,-1.8454871,0.6128815,0.13335837,0.78559226,0.28375047,0.81028396,-0.22977723,-0.7237857,1.6857818,0.509561,1.2747041,-1.902213,-0.5233789,1.1640823,-1.2161249,1.5746459,1.618402,-0.1257017,-0.45189327,0.21941599,1.5261029,0.2148943,-1.2782965,0.87880826,0.055001695,0.9844406,-0.20324627,-2.8322268,-0.3209339,-0.33477938,0.6969246,0.3011443,-0.9787141,1.6903338,1.1360927,-0.58386487,0.26703662,0.52846956,2.9326053,-0.16972595,-0.8636935,0.6924403,0.024309296,-0.36873165,1.1663344,-0.108403556,-0.95311296,-0.9590272,0.0822034,-0.25940424,-0.91215205,-1.8437147,1.152613,-0.9930276,-0.050513104,-0.112483285,-0.032333817,-0.15081748,-0.083487004,-1.0643237,1.939034,-0.014482598,-1.6800622,-0.25824916,-0.15790574,-0.1917895,-2.569876,0.7747697,-1.5408413,0.34215024,-0.06874411,-1.3181165,1.8203001,-2.104471,-0.5993394,-0.8646715,1.557215,-2.69426,0.21112908,1.5647161,0.8804989,0.5828577,0.29648244,0.14239098,0.3872025,0.5983569,-0.975884,-0.043937817,0.41530743,-0.7449577,0.2408017,0.6142519,-0.75696653,-0.13865414,1.2983172,-0.5393434,-0.9575251,-1.0924119,-1.4165851,-0.8175159,0.8285281]':::VECTOR, '[-0.8509671,-0.39572772,-1.3087205,-2.7056231,1.6440501,-0.098687515,2.9397376,-1.050523,-2.3024967,-0.91051465,-0.68626016,0.5792593,-0.1440204,-1.4477973,0.57904065,-2.0947418,-1.639247,1.7140226,-0.8038405,-0.2843887,0.1744933,-0.29590264,0.0893548,0.25184974,0.25302905,0.12228907,1.0554495,1.0004112,-0.79319453,0.6847999,-0.689461,0.63088405,0.2061573,-1.634775,0.31851372,1.2043169,-1.0832044,-0.37242836,-0.299444,-0.7475097,-1.2226658,1.237376,0.70345426,-0.31346062,0.9567167,1.4953667,-0.68929476,0.6758103,0.5544616,0.027488412,0.39362675,-0.8200002,-1.0167935,0.36208525,0.8931938,0.118986845,-0.3623396,-0.3161153,0.3489974,-1.3688966,0.65704787,1.5837113,-0.5114614,-0.29999048,1.2322351,-0.20374265,0.6268041,2.318069,0.860281,-1.5896809,0.4807182,0.06171702,2.1181967,-0.04838731,1.2150959,-0.96604925,0.7680559,0.39982623,0.047726274,0.4057191,-1.0023655,0.23746045,-0.2721395,-1.0390676,-0.64625496,0.14803818,0.91030186,-1.2397757,-0.67006224,2.078336,0.28630468,0.287378,2.588653,-0.15286633,1.6290185,0.065328546,-0.44696262,-0.7206925,-0.91900665,-0.7224439,-1.8026655,0.041717034,0.3325863,-0.37788492,0.094730966,0.83344054,-0.42909187,-0.8101653,-1.4912056,-0.08742666,-1.5246892,-0.701167,-0.60053617,0.47348344,-1.8823407,-0.1727989,1.48904,0.5608865,1.5880113,0.5362667,0.60114175,1.4396163,-0.5662212,1.6990653,1.4985507,-0.8215268,-0.11102759,-0.9682746,0.21722011,-0.2227615,-1.0412809,-0.054254636,-0.15481676,0.16481021,-1.9939976,-0.04008284,0.46435463,0.081453554,-1.1466643,-2.02749,-1.0585984,-1.0272297,-3.2058117,-0.40794998,1.118638,1.2734419,0.77359086,-1.0035794,0.8430635,-0.5945317,-1.3206725,-0.6480409,-1.445663,-0.019055054,0.6393802,-0.45174593,0.32036182,-0.43667194,-0.1168909,0.3281944,0.4215918,1.1518021,1.7429814,-0.991917,-1.8161211,0.20220268,0.35477307,-0.9162121,-1.0615945,0.23154618,0.3692767,0.8864294,1.5904549,0.4834773,-0.5029347,-0.9093378,-1.024459,-1.2762969,-0.11750197,-1.4489322,2.4078948,0.14504364,-0.54717255,-1.7590709,0.5017423,1.7125015,-0.39910284,-0.47256985,0.08424947,2.1430979,1.2349144,-1.4480627,1.70918,0.8790815,-0.4212714,-1.4839623,-0.029873025,-0.18235815,-0.69659483,0.30111784,0.27199566,-0.30136475,1.621072,-1.4819577,-0.81302327,0.002140476,0.30516508,0.962133,-0.2212365,1.0849499,-0.0056263884,-1.5388907,0.07425752,1.506651,-0.87566,-1.6994315,0.58126867,-1.4138108,-1.0808312,0.345949,0.15894958,0.5720124,-0.844951,-1.0163511,-0.32020503,-0.6415649,0.79869753,-0.35727623,0.695914,-0.9495843,0.5409711,0.88166773,-0.2239394,0.8051662,0.20225696,0.29192597,0.640204,-0.062455714,1.4306468,1.8068424,-0.7954108,-0.20859756,0.3364987,2.3158803,-1.0419377,0.0019160819,0.14591573,0.69441134,-0.1248293,1.8191302,1.2588949,0.5608391,0.55530167,-0.112121,-0.88101715,1.3543711,1.532715,2.4012694,0.8402445,1.457555,-1.1252315,0.20266622,0.70343125,-0.25385654,-1.2171836,0.8004958,-0.39125815,0.273149,-0.41583094,-0.26322132,0.2365034,0.10639854,-0.73130184,-0.23460442,-0.67724013,0.9505699,-0.7703006,0.5497526,-1.722491,-0.45270473,-0.6150793,-0.1458284,-0.73605406,1.2220669,-1.2696114,-0.30188993,0.4233886,1.2192382,1.146028,-0.58979243,1.5452697,-1.3253803,1.0621722,0.59406024,0.6847247,-0.19123918,-0.7624456,1.0125663,1.2301688,1.1003698,0.36962405,1.2099222,-0.34873208,0.6900965,0.01788366,-1.0462974,-1.6502082,0.076833524,0.36923707,0.62801695,-0.50995195,-0.9074796,-1.5199009,0.64762425,-0.2633981,-1.3903229,-0.7694892,0.0371829,0.08545162,0.28623646,0.17893463,-1.0618155,1.6373357,-0.9989676,-1.9415115,-0.51589847,-0.86255753,0.7093177,-0.10805876,1.0572503,1.3277466,-0.72375906,-0.36519796,-0.097631715,0.33497334,1.4848641,-0.40118733,-0.18761204,-0.42774782,0.44877094,-0.68620944,0.37910607,0.19682059,-1.3794268,-0.5980839,0.20807052,-0.52008605,0.38890314,2.1172433,-1.466018,1.6115445,-1.7915663,1.2186071,-1.1115056,-0.67910004,-0.54784864,-0.19689322,0.00034745608,-1.2405463,0.67638355,0.015386486,0.41343385,0.21693021,-0.6607455,-0.33621183,0.68988925,-0.20989226,-0.7053758,-0.44317788,-1.8448488,0.44474238,1.3328212,-1.2192707,-1.0310975,-0.90242434,0.13118745,0.84498733,1.2960826,-0.27288416,0.007990324,0.6827586,-2.908904,-0.25419182,-0.2864941,-0.8033586,0.8451655,0.20083098,-0.55924994,-0.075543426,-1.8686862,-0.0025614805,-0.21704806,1.4080305,0.2669784,-0.48657677,0.97269666,-0.848513,0.16387671,-0.5279352,0.09848904,-0.8036641,-0.49469748,0.4466464,2.304099,1.6459925,-0.8401403,-0.74567777,0.009805762,0.85615736,-1.3540696,-2.1764066,0.05013874,-0.24914613,-0.076702155,-0.11809351,-1.4018728,0.73647296,-1.4279813,-0.17830776,1.3095086,-0.75480396,-0.97661275,0.16979843,1.3660243,-0.15060194,-1.0907345,2.144816,-1.8756568,-0.27092865,0.83020717,0.107821666,0.86876726,-0.43811098,1.3751559,0.06467376,-0.73114055,0.29053012,-0.6411892,1.3467324,-1.4950147,-0.26659146,-0.38463953,0.8576507,-1.5863739,-0.47415265,-1.0526562,1.7645365,-1.4168847,0.5172464,-0.3964716,0.2176077,0.7432879,-0.82559735,0.5004248,-0.6886804,0.9606779,0.7481392,1.9663259,-0.072871834,2.8877196,-0.7848558,0.002845083,1.2294073,0.5198847,0.013006999,2.529814,-1.2716517,0.3286088,0.76881725,0.8459278,0.5646515,-1.0711907,0.88062304,-0.087828994,0.6759034,-0.34000567,0.9277508,-0.5283516,-1.681587,0.54967487,0.2758924,0.15000434,-0.64719117,0.44149223,-0.2228508,0.2822823,-1.1094996,-0.7074645,1.1691226,1.5864564,1.1385645,-0.99878854,-1.5268276,-0.50092584,0.561269,0.91861534,0.5339069,-0.6071325,-0.2911567,-0.9292395,0.73665345,-0.30317304,1.3114073,0.7736087,1.7014908,0.13331948,0.8729352,-1.676687,-0.9260156,0.3250878,-0.4854693,0.2351519,-0.85698694,0.19994482,-0.39131135,-0.05442252,1.3026482,1.5168245,-0.4449457,1.1264957,0.3931464,0.40245953,1.2750162,-0.30566782,0.31948736,1.537485,0.53593904,1.3010489,0.14245428,-0.7438509,0.3144278,-1.2748659,1.7810096,0.122978315,-1.5349141,-0.34403616,1.3088728,0.7509498,0.46996754,-0.51198584,-0.5142013,-0.37406877,-0.29643893,0.9509445,0.046821672,-1.9463565,0.08150081,0.15621436,1.2406411,0.07068587,0.18338807,-0.86582595,0.053101856,1.1394575,0.4342188,-0.9199707,-0.77295285,0.564877,-0.008336273,0.007610628,1.0831889,-0.16667897,1.0552309,0.5257187,0.44233268,-0.1724269,0.8123682,-0.7215925,1.3701358,-0.3990199,-0.8729184,-0.30832934,0.019836161,0.47675416,-1.4489987,0.2736833,2.4222114,-0.26364934,-0.039247524,1.2864094,-0.31989422,-1.3797286,-1.9719399,-0.77930135,1.3476596,-0.903154,0.45366284,1.2053728,0.5104344,-1.0338237,-1.5632803,-0.49129456,0.10953898,-1.6362438,0.40649557,-0.4904259,-0.1786302,-0.90663743,-0.5909127,-0.74152386,-0.627753,-1.681177,-0.041357666,0.3431759,-0.25114197,1.0327251,-0.24488111,-0.22143921,1.301557,0.30869925,0.22591157,0.6020532,-1.5987833,-1.4590298,-0.83897555,0.9987309,-0.14574048,0.07964699,-1.0743514,0.39580637,1.6698183,0.9238673,0.0712481,0.5837933,-0.44250876,-0.0051525496,-0.5442493,-0.21639387,0.77262175,1.0813144,-0.8684888,0.00023830448,-0.7927341,-0.9718135,-0.89498466,0.25477841,-0.6643366,-1.6754569,-0.036036387,-1.3205796,0.69441867,0.2252009,-1.866597,1.063131,-1.3789617,-0.4455762,-0.7734726,-1.7248772,0.67501,1.952624,0.18738447,-0.21730228,-0.68177897,0.60381585,0.9654725,0.6496032,-1.2292492,0.56127864,-0.4475632,-1.5203053,-0.87460333,0.73768944,-1.0594385,-0.66809964,0.7588471,0.31637847,1.0721769,1.4902092,0.048300724,2.47762,-0.82182777,0.5184639,2.1333218,2.2101538,-1.4499443,0.6617959,0.29903594,1.143269,1.5394685,-1.3373752,-1.2595744]':::VECTOR) 
    rsg_test.go:125: Last executed (26): SELECT int2out(25323:::INT8) 
    rsg_test.go:125: Last executed (27): SELECT int2recv('\x':::BYTES) 
    rsg_test.go:125: Last executed (28): SELECT int2send(0:::INT8) 
    rsg_test.go:125: Last executed (29): SELECT int2vectorin('[1.3980839,-2.6859815,-1.9358569,-0.24591804,-0.8044362,0.4967859,1.1542233,0.47504503,-0.6797977,1.6787724,0.15646456,0.00494562,1.056525,0.26716632,-0.93290335,-0.07335383,-0.709484,-1.3841833,-0.7958262,-0.39271325,0.30679548,0.98347795,0.6462224,-1.4927729,-0.885026,0.8890484,0.45085418,-0.7119107,-0.4489533,0.52801377,-0.13140687,-0.5718709,0.20061715,-0.21044451,-0.068165466,-0.85655904,0.57148385,0.44784552,-0.5899629,-0.866643,1.6903473,-0.5457323,-0.2538318,-1.4276857,-0.20332856,0.6513959,1.7865734,-0.82481116,-0.35531747,-0.5999062,-0.18961404,0.68254095,0.9962501,0.84184587,-0.38383016,0.54577607,0.36067602,0.37567416,-1.2432535,1.704098,0.47759792,1.9417281,-1.4547004,-0.95009905,-1.7406216,-0.21369512,-0.872925,0.22132458,2.5706887,0.43818772,1.1408705,-0.12115889,-1.1312522,0.043454114,0.5564246,-0.29553568,1.9890164,0.93884146,-0.6706253,1.4126412,2.0239806,-0.51075006,-0.7694651,-1.0531001,-1.0792032,-1.0148962,0.73419005,-0.5050806,-0.028715067,-0.47059697,0.60900533,-0.42519674,-0.44364202,0.49730808,-2.7795694,0.5868454,-0.6562571,-0.92329425,-0.122437455,1.3868892,0.23597541,0.93338007,0.23847434,-1.4589494,-0.87346953,-1.6455576,1.4498996,-0.20281313,-0.9944386,-1.6770977,1.1940279,-0.19467072,1.6493756,0.026549792,-0.710286,-0.64596754,0.19543049,0.05657051,-1.4819407,-0.57096326,-0.25214604,0.5595696,-0.39460263,-0.060542494,-0.084179714,0.7088299,0.15704925,-1.3751984,-0.29749134,0.064812675,0.5637,-0.70206577,-2.4263346,-0.3297625,0.08963396,0.96319026,-0.45801324,-0.55923134,0.12173359,-1.3249398,-1.178099,-0.40419334,2.6278384,0.065315,-0.062438443,-0.008325582,-1.4094324,-1.5627519,-1.1995813,-0.99899954,-1.226083,-1.4418429,-0.28450888,-1.2523046,1.3765978,1.3673489,1.4148777,-0.019734088,0.037127838,0.96140444,1.1269299,-0.53226507,0.29511175,0.9306578,0.09933791,1.5447111,0.87869525,-0.9484907,-0.68905205,0.74423724,1.2435967,0.07365138,1.4544972,0.93946207,0.26926148,0.7220534,-0.31360024,1.4421295,0.086441904,-0.21568732,-0.22450288,-0.83485633,0.024989156,0.73254305,-0.30888,-0.59849364,1.0202101,-0.10536436,0.22176059,-2.144294,1.0401645,-0.6476289,0.65926063,0.9255403,0.4709049,0.04787461,0.1116926,-0.08658407,0.69721156,-1.0688337,-0.7505326,-0.43239275,-1.1296512,-0.669209,-0.11407125,-0.6369365,-0.4766193,1.1482533,0.8813299,0.11453585,-0.4711768,1.6075552,0.4430419,0.8912517,0.6607314,-1.1275296,0.33801633,0.8280451,-1.2108725,1.1588118,1.0744748,-0.74166316,0.4943223,-0.24967305,-1.0623335,-0.5056193,0.65511715,-0.091595314,-0.045032497,0.9871992,-1.9886209,0.7586261,-0.2660212,0.5776891,-0.46325573,2.3153133,0.52805436,0.34281078,0.30875918,-2.0349753,-0.019527448,-0.77515316,0.90264386,-0.84466857,-0.49252474,-1.9052432,0.89408827,0.06765829,-0.8031032,0.6678201,0.014418279,0.019684808,2.1315267,-1.4561249,1.0252175,0.16364053,-0.3729517,0.6127868,0.87557554,-1.9001695,1.0921581,0.98871595,-0.12923627,-0.30322132,-0.24056412,-2.2802083,-1.7919352,0.29222447,0.8939764,-0.8208018,2.331998,-1.583284,-0.57532877,-0.9888415,-0.8968408,0.6159438,-0.72138137,-0.9302884,-1.3330436,0.6432418,-0.3931834,-0.09498415,-0.29673505,0.8769721,0.48937127,1.3721052,0.5930096,0.17645827,-1.2605335,0.7468136,0.61222386,-0.34248817,-0.3477384,2.7170315,-0.33629546,1.5769814,0.46135384,-0.13056134,0.9370116,0.59041005,-1.3820686,-0.11505311,0.82852906,-1.1079814,-1.3775755,0.07351225,0.6326126,1.0792402,0.011208335,-0.9126617,2.8210337,0.6409528,-1.7256247,0.2724372,-0.5052399,-0.5186863,2.4897711,0.29530624,0.6349867,0.2726759,-1.1733298,-0.2016441,-0.53753835,0.91655207,-0.591592,0.18585049,-0.3727239,-0.20103274,-1.3994026,-0.57856065,-0.10533071,0.24017021,-0.3027738,1.5676451,0.99837816,-1.5030915,1.4632223,0.7558074,0.678275,-0.33660173,0.017348614,-0.045516483,-0.6461061,0.7280593,-0.04678662,0.61498415,0.18658102,-0.5135401,0.18818262,0.27080607,0.99391514,1.7280881,1.557191,-0.38049427,0.4842516,-0.3555741,-0.70200086,0.60749376,-0.3774898,-0.37408608,-0.4068836,-0.85238326,-0.23855995,0.35459638,0.8393759,0.8642414,-0.38783246,-0.09786787,-0.07325545,0.7365714,0.65962535,1.0313308,1.2823122,0.41350996,1.281432,0.8805654,-1.5647608,-0.044452876,-0.85382706,-0.10878123,0.4129905,-0.72477096,0.36698177,-0.10251637,0.5650768,1.1424549,-0.3080355,1.857679,-0.10961712,-0.36046964,0.62379783,-0.2038596,-0.60816467,-0.41411003,1.1521572,1.1560247,0.26617476,-0.12509166,0.47333178,0.5632381,-1.84057,-0.1870824,0.52770996,-1.700677,1.76169,-0.1828054,-0.47038192,-0.10072024,1.1356742,-0.83803177,0.43829378,-1.2874998,0.09415275,1.6986226,-0.07584283,1.8259385,0.14892694,1.6992639,1.3572222,-1.7397113,2.0146034,0.5320663,-1.7015823,0.852951,-0.26032937,0.1537733,0.31802315,1.0430323,-0.2068522,0.07171494,0.91191983,-1.0967503,0.86107796,-1.3823577,1.188657,1.828664,0.6028341,0.609503,0.6682574,-1.1911191,-0.72891855,-0.17927009,0.118493564,-1.222974,0.8315723,2.2417147,-1.1481243,0.679279,0.9261656,-0.06946741,0.95968586,0.0920761,0.5836281,-1.2496496,0.08044761,-1.0329698,1.592674,0.53716266,-1.0098556,-0.55510575,-1.2707329,-0.84643227,-1.4526932,0.08176199,-0.8747249,-0.452881,1.8834599,1.7055856,-0.9970132,0.31717205,-1.5467446,-1.7346781,1.7838869,0.060822368,-0.3340201,0.11751762,1.4358233,-0.33378506,1.1768348,-0.23400472,1.3475708,-0.10698639,0.37868676,-0.3090439,-0.053626183,-1.9574782,0.36948395,1.2535663,-0.44980252,0.5912753,-1.2310368,0.31102836,-0.35626987,-0.53815496,-0.903423,-0.35847408,-2.3144753,-0.026493408,0.34469894,1.1296015,-1.9360001,-1.4611754,-0.6006824,-0.71915996,0.6312705,-0.7633439,0.99908316,-1.4606966,0.20109533,0.46139127,-2.0095448,-0.47090673,-1.0117183,1.205119,0.82707536,-1.2881485,1.0787964,-0.09488289,0.08331342,-0.13627073,-0.19711955,-0.5676408,-0.38633266,-2.2782972,-0.19073635,0.06147462,-1.5810992,-0.54124695,-0.9632759,0.42824435,-0.36511287,-0.322002,0.26509598,-2.373545,0.29668093,0.26412657,-1.1960701,-0.06363523,0.22137092,-1.3595884,0.10038225,0.6969692,0.32115057,1.0926441,-0.2439727,1.3197219,-0.74819416,-1.1625893,-1.0327979,-0.53751487,-0.7121215,0.6000483,-1.1772556,-1.508263,-1.0876685,-0.90819466,-0.33485812,-0.37074816,1.002056,0.7674832,0.83961487,0.1944496,1.904469,-0.18833748,-0.29243523,1.0501052,-0.3884204,-0.7010681,-0.26406202,-0.43286878,0.8698453,-2.3044176,0.5919465,-0.7438833,-0.82017076,0.60785836,-1.7764887,-1.3594073,-0.2153989,1.2693225,-0.91771376,-1.7734658,-2.1708007,-1.4750444,-0.27331862,4.3643174,0.012385585,-1.371932,0.65346104,-0.25836065,-1.584898,0.6358871,0.18833274,-0.32568172,-0.30530626,0.4010873,0.17823647,0.8495061,-0.080964044,1.4924406,1.0931681,-0.9671254,-1.1267449,0.42647925,0.34066424,0.9872883,-0.8736891,0.040302772,1.2364182,0.2677601,0.7161073,0.13000743,-0.5784453,0.383588,-1.3957778,0.9253514,-1.1449827,-1.7076571,-0.9976961,1.2037457,1.4748529,0.110300176,0.4369196,-0.66987437,-1.4937488,0.043085,-0.5736406,-0.83765215,0.18429208,-0.36389014,1.0635347,-1.0908916,1.0868852,0.51130325,-0.7194534,-0.30342585,0.12440067,1.4220673,-0.009651958,0.40472463,2.0249867,-0.1841805,-0.7580737,0.20878184,0.7905637,0.26960474,-0.7824516,-1.4683561,-0.05627418,-0.14295688,-3.156748,1.0224208,-0.73293597,0.5425612,-0.99392027,-0.067884065,0.43420857,-0.32483348,-0.3625923,0.69787055,1.4829895,0.16793166,-0.36019683,0.8314186,-0.32300714,-1.6812077,0.7619798,0.3582946,0.5543094,1.8496706,0.34406787,-1.3819766,0.59364456,0.6445361,1.3414453,-0.40649053,-0.46480393,0.058712766,-1.4579836,0.9662863]':::VECTOR) 
    rsg_test.go:125: Last executed (30): SELECT int2vectorout(ARRAY[15904:::INT8,(-8688):::INT8,(-8133):::INT8,(-16453):::INT8]) 
    rsg_test.go:125: Last executed (31): SELECT int2vectorrecv(1006698114:::REGPROCEDURE) 
    rsg_test.go:125: Last executed (32): SELECT int2vectorsend(ARRAY[8233:::INT8,6054:::INT8,(-15879):::INT8]) 
    rsg_test.go:125: Last executed (33): SELECT int4in((SELECT NULL)::anyelement) 
    rsg_test.go:125: Last executed (34): SELECT int4out((-1175610496):::INT8) 
    rsg_test.go:125: Last executed (35): SELECT int4recv('m':::STRING) 
    rsg_test.go:125: Last executed (36): SELECT int4send((-761843284):::INT8) 
    rsg_test.go:125: Last executed (37): SELECT int8in(0:::REGPROC) 
    rsg_test.go:125: Last executed (38): SELECT int8out(112276282954093029:::INT8) 
    rsg_test.go:125: Last executed (39): SELECT int8recv('u':::REFCURSOR) 
    rsg_test.go:125: Last executed (40): SELECT int8send((-32768):::INT8) 
    rsg_test.go:125: Last executed (41): SELECT interval_in((SELECT NULL)::anyelement) 
    rsg_test.go:125: Last executed (42): SELECT interval_out('71 years 9 mons 917 days 06:42:27.05824':::INTERVAL) 
    rsg_test.go:125: Last executed (43): SELECT interval_recv(B'00000001011001110110100010001110001') 
    rsg_test.go:125: Last executed (44): SELECT interval_send(NULL) 
    rsg_test.go:125: Last executed (45): SELECT isnan('NaN':::FLOAT8) 
    rsg_test.go:125: Last executed (46): SELECT isnan(2.795861562157888081E+19:::DECIMAL) 
    rsg_test.go:125: Last executed (47): SELECT json_agg(2565441727:::REGPROC) 
    rsg_test.go:125: Last executed (48): SELECT json_array_elements(NULL::jsonb) 
    rsg_test.go:125: Last executed (49): SELECT json_array_elements_text('{"B9jvNfiT": [null], "GvzamiB": "WTmfN3qIU", "a": [3.226594495319304, "foobar"], "b": true, "foobar": null}':::JSONB) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions3769182897
--- FAIL: TestRandomSyntaxFunctions (301.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/testdata/logic_test/statement_hint_builtins
+++ b/pkg/sql/logictest/testdata/logic_test/statement_hint_builtins
@@ -943,3 +943,11 @@ statement hints count: 1
       missing stats
       table: xy@xy_pkey
       spans: FULL SCAN
+
+# Test error handling for crdb_internal.inject_hint.
+
+statement error pq: could not parse statement fingerprint as a single SQL statement
+SELECT crdb_internal.inject_hint('', '')
+
+statement error pq: could not parse hint donor statement as a single SQL statement
+SELECT crdb_internal.inject_hint('SELECT 1', 'SELECT 2; SELECT 3')

--- a/pkg/sql/sem/builtins/builtins.go
+++ b/pkg/sql/sem/builtins/builtins.go
@@ -9677,18 +9677,32 @@ WHERE object_id = table_descriptor_id
 				fingerprintFlags := tree.FmtFlags(tree.QueryFormattingForFingerprintsMask.Get(
 					&evalCtx.Settings.SV,
 				))
-				targetStmt, err := parserutils.ParseOne(stmtFingerprint)
+				stmts, err := parserutils.Parse(stmtFingerprint)
 				if err != nil {
 					return nil, pgerror.Wrap(
 						err, pgcode.InvalidParameterValue, "could not parse statement fingerprint",
 					)
 				}
-				donorStmt, err := parserutils.ParseOne(donorSQL)
+				if len(stmts) != 1 {
+					return nil, pgerror.New(
+						pgcode.InvalidParameterValue,
+						"could not parse statement fingerprint as a single SQL statement",
+					)
+				}
+				targetStmt := stmts[0]
+				stmts, err = parserutils.Parse(donorSQL)
 				if err != nil {
 					return nil, pgerror.Wrap(
 						err, pgcode.InvalidParameterValue, "could not parse hint donor statement",
 					)
 				}
+				if len(stmts) != 1 {
+					return nil, pgerror.New(
+						pgcode.InvalidParameterValue,
+						"could not parse hint donor statement as a single SQL statement",
+					)
+				}
+				donorStmt := stmts[0]
 				donor, err := tree.NewHintInjectionDonor(donorStmt.AST, fingerprintFlags)
 				if err != nil {
 					return nil, errors.NewAssertionErrorWithWrappedErrf(err, "error while creating donor")

